src/Basic/Isomorphism.lidr

```diff
@@ -61,6 +62,35 @@ along with this program.  If not, see <https://www.gnu.org/licenses/>.
 >   inverse
 >   (MkInverseMorphisms leftInverse rightInverse)
 >
+> isomorphismEq :
+>      {cat : Category}
+>   -> {a, b : obj cat}
+>   -> (hom : mor cat a b)
```

Comment:
what do you say about making this `hom` argument implicit?

Reply:
Can do! I mean to email you about implicits at some point.